server/core/src/https/views/reset.rs

```diff
@@ -744,21 +762,41 @@ fn get_cu_partial_response(cu_status: CUStatus) -> Response {
         .into_response()
 }
 
-fn get_cu_response(domain: String, cu_status: CUStatus) -> Response {
+fn get_cu_response(domain: String, cu_status: CUStatus, uri: Uri, is_logged_in: bool) -> Response {
     let spn = cu_status.spn.clone();
     let displayname = cu_status.displayname.clone();
     let (username, _domain) = spn.split_once('@').unwrap_or(("", &spn));
     let names = format!("{} ({})", displayname, username);
     let credentials_update_partial = get_cu_partial(cu_status);
-    (
-        HxPushUrl(Uri::from_static("/ui/reset")),
-        HtmlTemplate(CredResetView {
+
+    if is_logged_in {
+        let cred_status_view = CredStatusView {
+            uri,
             domain,
             names,
             credentials_update_partial,
-        }),
-    )
-        .into_response()
+            posix_enabled: false,
```

Comment:
add a TODO so I remember to fill this in :)

Reply:
you mean like a comment above this right?

Reply:
yeah, // TODO: fill in posix enabled